js/models/serverConfigMd.js

```diff
@@ -82,7 +82,9 @@ module.exports = Backbone.Model.extend({
       }
     }
 
-    return Object.keys(err).length && err || undefined;
+    if (Object.keys(err).length > 0) {
```

Comment:
Minor, but the `> 0` is superfluous. This could have just been:

```
if (Object.keys(err).length) {
...
```

but, it's so minor, we won't hold up the merge for it.